src/CONST.ts

```diff
@@ -1264,7 +1264,7 @@ const CONST = {
         CARD_NUMBER: /^[0-9]{15,16}$/,
         CARD_SECURITY_CODE: /^[0-9]{3,4}$/,
         CARD_EXPIRATION_DATE: /^(0[1-9]|1[0-2])([^0-9])?([0-9]{4}|([0-9]{2}))$/,
-        ROOM_NAME: /^#[a-z0-9à-ÿ-]{1,80}$/,
+        ROOM_NAME: /^#[\p{Ll}0-9-]{1,80}$/u,
```

Comment:
@0xmiroslav `\p{Ll}` will include the lowercase letters from any language, e.g., Greek and Cyrillic alphabets will also be valid.

And if we want to accept only Latin lowercase characters explicitly, we must adjust the regex accordingly.
Then We have to use this regex `/^#(?:((?=\p{Script=Latin})\p{Ll})|[0-9-]){1,80}$/u` that will check the Latin script lowercase alphabets only, above regex will create positive lookahead, which asserts that the following character(s) are Latin script characters which following by lowercase letter.

Reply:
Yeah I think that's fine to accept other language alphabets